graphite/graphite_tcp.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // TCPServer processes Graphite data received over TCP connections.
 type TCPServer struct {
-	server Server
+	server SeriesWriter
```

Comment:
I'm not sure I like how these names now line up.  I understand that we introduced another Server interface above.  Might be better as `GraphiteServer` and then `Server` or something along that line.  I can see this be confusing for the next developer.